### PR TITLE
[prometheus] Fix alerts-receiver CVE

### PR DESCRIPTION
**Pull request by scaps1**
## Description
Fix of image vulnerabilities

## Why do we need it, and what problem does it solve?
To improve security of deckhouse

## What is the expected result?
Fixed CVEs

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: prometheus
type: fix
summary: Fix alerts-receiver CVE 
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: modules/300-prometheus/images/alerts-receiver/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+  add: /app/alerts-receiver
+  to: /alerts-receiver
+  before: setup
+docker:
+  ENTRYPOINT: [ "/alerts-receiver" ]
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/300-prometheus/images/alerts-receiver/werf.inc.yaml:13]:
```suggestion
image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact
fromImage: common/src-artifact
final: false
```

File: modules/300-prometheus/images/alerts-receiver/werf.inc.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+git:
+- add: /{{ $.ModulePath }}modules/300-{{ $.ModuleName }}/images/{{ $.ImageName }}/src
+  to: /src
+  stageDependencies:
+    install:
+    - '**/*'
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+from: {{ $.Images.BASE_GOLANG_23_ALPINE }}
```
Comment (RomanenkoDenys) [inline modules/300-prometheus/images/alerts-receiver/werf.inc.yaml:22]:
```suggestion
image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
from: {{ $.Images.BASE_GOLANG_23_ALPINE }}
final: false
```

File: modules/300-prometheus/images/alerts-receiver/werf.inc.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+image: {{ .ModuleName }}/{{ .ImageName }}
+fromImage: common/distroless
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
```
Comment (RomanenkoDenys) [inline modules/300-prometheus/images/alerts-receiver/werf.inc.yaml:5]:
```suggestion
- image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

**Pull request merged**